cachelib/cachebench/util/CacheConfig.h

```diff
@@ -41,6 +41,28 @@ class CacheMonitorFactory {
   virtual std::unique_ptr<CacheMonitor> create(Lru2QAllocator& cache) = 0;
 };
 
+struct MemoryTierConfig : public JSONConfig {
```

Comment:
Please add some documentation for this struct.

Reply:
Done